src/pytest_check/check_raises.py

```diff
@@ -39,23 +48,23 @@ def raises(expected_exception, *args, **kwargs):
     __tracebackhide__ = True
 
     if isinstance(expected_exception, type):
-        excepted_exceptions = (expected_exception,)
+        expected_exceptions: Iterable[type] = (expected_exception,)
     else:
-        excepted_exceptions = expected_exception
+        expected_exceptions = expected_exception
 
     assert all(
         isinstance(exc, type) or issubclass(exc, BaseException)
-        for exc in excepted_exceptions
+        for exc in expected_exceptions
     )
 
     msg = kwargs.pop("msg", None)
     if not args:
         assert not kwargs, f"Unexpected kwargs for pytest_check.raises: {kwargs}"
-        return CheckRaisesContext(expected_exception, msg=msg)
+        return CheckRaisesContext(*expected_exceptions, msg=msg)
```

Comment:
The typing on CheckRaisesContext could probably be expanded to be `_TypeTuple` (like in `check_functions.py`), but the var-arg seemed a bit cleaner.